alphagov/hand-signals
# Remove Hand Raise signal, update design and tidy-up

## Code changes

- **remove Raise hand signal**
Avoid duplication with the built-in Raise hand functionality in Google Meet

- replace individual button components that had very similar but slightly different logic with one `Reaction` component where we pass the type and send the appropriate message and emoji
- cleanup

## Design change
### Before
![Screenshot 2021-07-06 at 18 24 46](https://user-images.githubusercontent.com/3758555/124642190-77652280-de87-11eb-8ba6-7ea8e58fae3c.png)


### After
![Screenshot 2021-07-06 at 18 05 12](https://user-images.githubusercontent.com/3758555/124642287-96fc4b00-de87-11eb-9560-7701bdedcb26.png)

![Screenshot 2021-07-06 at 18 05 20](https://user-images.githubusercontent.com/3758555/124641897-19d0d600-de87-11eb-8103-233a6cf38021.png)


## How to test
- remove installed extension
- build and load local - see https://github.com/alphagov/hand-signals#how-to-test-the-extension-locally
